stock_picking_import_serial_number/models/res_config_settings.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2022 Tecnativa - Sergio Teruel
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class ResConfigSettings(models.TransientModel):
+    _inherit = "res.config.settings"
+
+    default_search_product_by_field = fields.Selection(
```

Comment:
Please use more specific field names in the wizard, or there's a high chance of collapsing with other modules. For example: `default_import_s_n_search_product_by_field`. I know that this seems a bit awkward on the wizard side (unless there's a key here to specify the field name, having the possibility of putting a different name).